### ISSUE TITLE
 Error: Invalid proxy's attribute assignment - additional-trust-bundle

### ISSUE THREAD

**Issue by vamsinm** [terraform-redhat/terraform-provider-rhcs#361]
we are able to configure additonal-trust-bundle with out needing to configure http-proxy or https-proxy using command `rosa edit cluster -c <cluster_name> --additional-trust-bundle-file "<certfile path>". when using terraform it is forcing to provide http-proxy or https-proxy. 

```
╷
│ Error: Invalid proxy's attribute assignment
│ 
│   with module.cluster.module.cluster.rhcs_cluster_rosa_classic.rosa_cluster,
│   on .terraform/modules/cluster/modules/cluster/rosa_cluster.tf line 1, in resource "rhcs_cluster_rosa_classic" "rosa_cluster":
│    1: resource "rhcs_cluster_rosa_classic" "rosa_cluster" {
│ 
│ Expected at least one of the following: http-proxy, https-proxy
```

**Comment by nirarg** [comment]
Hi @vamsinm

Thank you for adding sharing this issue
Can you please share the the following details:
1. RHCS terraform provider version
2. Terraform CLI version
3. The resource you used (in main.tf file)
4. In case you added this attribute as a variable, how did you assign it? (tfvar file, environment variable, etc)

**Comment by seanmalloy** [comment]
Jira ticket: [OCM-4621](https://issues.redhat.com/browse/OCM-4621)

**Comment by seanmalloy** [comment]
It's my understanding that this bug is fixed by #406 and  #407.

**Comment by bardielle** [comment]
@seanmalloy The v1.4.2 would be released on Dec 06

**Comment by seanmalloy** [comment]
@bardielle thanks for the update. It looks like you've created a follow up fix in #415.

**Comment by bardielle** [comment]
Version v1.4.2 was released today.
Please update/ close the issue.

**Comment by seanmalloy** [comment]
I was able to confirm with @vamsinm that this issue has been fixed in release v1.4.2.  Please close this issue.

Thanks.
